tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/ResponseHandlerTests.cs

```diff
@@ -38,5 +41,80 @@ public async Task HandleUserResponse()
+            bool hasChanges = deltaResponse.AdditionalData.TryGetValue("changes", out object changes);
+
+            // Can't test against the actual model, see issue #57 for more info.
+            var serializer = new Serializer();
+            List<string> changeList = serializer.DeserializeObject<List<string>>(changes.ToString());
+            
+            // Assert
+            Assert.True(hasChanges);
+            Assert.Equal("value[0]['@odata.type']", changeList[0]);
+            Assert.Equal("value[0]['@odata.etag']", changeList[1]);
+            Assert.Equal("value[0].subject", changeList[2]);
+            Assert.Equal("value[0].body", changeList[3]);
+            Assert.Equal("value[0].body.contentType", changeList[4]);
+            Assert.Equal("value[0].body.content", changeList[5]);
+            Assert.Equal("value[0].start", changeList[6]);
+            Assert.Equal("value[0].start.dateTime", changeList[7]);
+            Assert.Equal("value[0].start.timeZone", changeList[8]);
+            Assert.Equal("value[0].end", changeList[9]);
+            Assert.Equal("value[0].end.dateTime", changeList[10]);
+            Assert.Equal("value[0].end.timeZone", changeList[11]);
+            Assert.Equal("value[0].organizer", changeList[12]);
+            Assert.Equal("value[0].organizer.emailAddress", changeList[13]);
+            Assert.Equal("value[0].organizer.emailAddress.name", changeList[14]);
+            Assert.Equal("value[0].organizer.emailAddress.address", changeList[15]);
+            Assert.Equal("value[0].organizer.customProp", changeList[16]);
+            Assert.Equal("value[0].id", changeList[17]);
+            Assert.Equal("value[1]['@odata.type']", changeList[18]);
+        }
+
+        [Fact]
+        public async Task HandleEventDeltaResponseWithEmptyCollectionPage()
+        {
+            // Arrange
+            var deltaResponseHandler = new DeltaResponseHandler();
+
+            // Empty result
+            var testString = "{\"@odata.context\":\"https://graph.microsoft.com/v1.0/$metadata#Collection(event)\",\"@odata.nextLink\":\"https://graph.microsoft.com/v1.0/me/calendarView/delta?$skiptoken=R0usmci39OQxqJrxK4\",\"value\":[]}";
+
+            var hrm = new HttpResponseMessage()
+            {
+                Content = new StringContent(testString,
+                                            Encoding.UTF8,
+                                            "application/json")
+            };
+
+            // Act
+            var deltaResponse = await deltaResponseHandler.HandleResponse<EventDeltaCollectionResponse>(hrm);
+            bool hasChanges = deltaResponse.AdditionalData.TryGetValue("changes", out object changes);
+
+            var serializer = new Serializer();
+            List<string> changeList = serializer.DeserializeObject<List<string>>(changes.ToString());
+
+            // Assert
+            Assert.True(hasChanges);
```

Comment:
If the "changeList" is indeed empty, should we have changes? Or should `hasChanges` be false?